src/python/pants/backend/python/typecheck/pyright/rules_integration_test.py

```diff
@@ -92,29 +92,49 @@ def add(x: int, y: int) -> int:
     """
 )
 
+PYRIGHT_VERSION = "1.1.365"
```

Comment:
Seems unrelated to this MR?

Reply:
See explanation above, I'll take this out